dashboard/views.py

```diff
@@ -334,7 +347,7 @@ def grade_distribution(request, course_id=0):
     logger.debug(f"Grades distribution: {grades}")
     BinningGrade = find_binning_grade_value(grades)
     if BinningGrade is not None and not BinningGrade.binning_all:
-        df['current_grade'] = df['current_grade'].replace(df['current_grade'].head(BinningGrade.index),
+        df['current_grade'] = df['current_grade'].replace(df['current_grade'].head(BinningGrade.bin_index),
```

Comment:
Maybe we can change the name of this variable `BinningGrade` variable here? Seems confusing since its type is `BinningGrade`, but it's an instance of it. It also doesn't follow PEP for variable naming. `binning_grade` would probably be fine.